src/main/java/fr/xephi/authme/output/Log4JFilter.java

```diff
@@ -16,6 +16,12 @@
 
     private static final long serialVersionUID = -5594073755007974254L;
 
+    private LogFilterService filterService;
+
+    public Log4JFilter(LogFilterService filterService) {
+        this.filterService = filterService;
```

Comment:
> I think that using the ConsoleFilter(LogFilterService) constructor is a bid dirty, any suggestion?

I don't really have a better suggestion. In theory we could put `@Inject` on the constructor and have a Log4JFilter instance injected into OnstartupTasks, but given the nature of how it is set up (in OnStartupTasks we have compatibility checks in case a certain log class doesn't exist) I wouldn't recommend doing that. I think your current solution works out nicely :)